src/routes/recipe.ts

```diff
@@ -38,4 +40,17 @@ router.get(
   validate,
   getRecipesLikeByUserId
 );
+router.delete("/recipe/:id", ({ params }, res) => {
+  RecipeModel.findOneAndDelete({
```

Comment:
Good start. This deletes the recipe by id from the database, but it doesn't take care of updating the user. 
We already created a controller that takes care of all of this functionality so feel free to use it here instead. We have access to the user's id via `req.user.id`

```suggestion
  RecipeModel.deleteRecipeById({
```